tests/helpers.bash

```diff
@@ -2,7 +2,7 @@ DUMP_FILE=$(mktemp)
 MONITOR_PID=""
 
 function monitor_start {
-	sudo cilium monitor > $DUMP_FILE &
```

Comment:
The issue is that if you run the script with sudo it will fail to kill the monitor. `sudo sudo kill` doesn't work as expected. I'd rather just have the entire script be run with required privileges.